server/src/main/java/org/elasticsearch/index/flush/FlushStats.java

```diff
@@ -85,6 +96,7 @@ public TimeValue getTotalTime() {
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject(Fields.FLUSH);
         builder.field(Fields.TOTAL, total);
+        builder.field(Fields.PERIODIC, periodic);
```

Comment:
can we have a rest test that checks that this is there?

Reply:
@s1monw 
I had a test but I could not verify the periodic value as the periodic flush is executed async. Do you have any suggestion for this? Or just check its presence (eg. 0 is ok)?

Reply:
I used the analogue of `gt: { periodic: 0}` when I did something similar.

Reply:
yeah ++

Reply:
I will go this suggestion. Thanks @DaveCTurner and @s1monw.